contents/blog/mixpanel-alternatives.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "The most popular Mixpanel alternatives, compared"
+date: 2023-09-26
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/posthog-alternatives/posthog-alternatives.jpeg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+INTRO INTRO INTRO 
+
+
+### What is PostHog?
+
+PostHog is an all-in-one platform that combines product analytics, session replay, feature flags, A/B testing, and surveys into one product. It can also replace customer data platforms (CDP), such as Segment.
+ 
+### Key features
+
+- **Product analytics**, including funnel insights, user paths, retention analysis, custom trends, and dynamic user cohorts. Power users can create custom insights using SQL.
+
+- **Session replays** with event timelines, console logs, and network activity. You can also download recordings to save them beyond the standard 90-day retention policy.
+
+- **Feature flags** with local evaluation (for faster performance) and JSON payloads, so you can make changes to your product without deploying new code. 
+
+- **A/B tests** with up to 9 test variations, primary and secondary metrics. Automatically calculates recommended duration, sample size, and statistical significance.
+
+- **Surveys**, including open text, link with custom CTA, ratings (for NPS and PMF surveys), single and multiple choice. Target users based on user properties, URL, or using existing feature flags. 
+
+### How popular is PostHog?
+
+According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent consultant:
+
+- PostHog is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. It's the fastest growing product analytics tool on the market.
+
+- Mixpanel is deployed by 5,218 (0.5%) of the top 1 million websites in 2023, down -17% from 6,295 the previous year.
+
+Data correct as of July 2023.
+
+### Why do companies use PostHog?
+
+According to [reviews on G2.com](https://www.g2.com/products/mixpanel/reviews), [summarized by Senja](https://senja.io/tools/review-mining/share?s=NOmOO5mS), companies use PostHog because:
```

Comment:
Personally, I don't like the addition of Senja here. Better to say that reviews show that it has these three main perks, among others?

I'd also add a section on WHO uses PostHog. Emphasize ICP, and I can back it up with G2 data if needed. 

Of course, if you're going to bring G2 into the mix then it raises the question of whether you want to use their reports, badges, etc. If not, is G2 the right platform to bring up and would we be better pointing to the case studies for social proof instead? Case studies are probably a good addition here anyway.

Reply:
- RE: Senja, I think it's useful to show we're not just making up our own points here. I can probably de-emphasize so it's not so prominent, though.

- Will look at ways to work customer stories in, but I'm trying to keep the top-level stuff neutrally sourced.